devito/arch/archinfo.py

```diff
@@ -31,6 +31,8 @@
            'POWER8', 'POWER9',
            # Generic GPUs
            'AMDGPUX', 'NVIDIAX', 'INTELGPUX',
+           # Nvidia GPUs
+           'AMPERE',
```

Comment:
Why not Volta/Hopper/Blackwell too?

Reply:
unnecessary. I only need to import AMPERE in PRO for testing